configs/steam-input/emulationstation-de_controller_config.vdf

```diff
@@ -989,19 +989,36 @@
 					{
 						"bindings"
 						{
-							"binding"		"controller_action CHANGE_PRESET 14 1 1, ares, EmuDeck_ControlConfig.png, #665200 #FFFFFF"
+							"binding"		"key_press KEYPAD_PERIOD, How to Use: Long press an emulator icon or hotkey to activate. Emulators not found here use combo hotkeys., ghost_030_inv_0312.png, #420058 #FFFFFF"
```

Comment:
Long press an emulator icon or hotkey to activate. Emulators not found here use combo hotkeys
Long press to activate. Other emulators use combo hotkeys.

Seems shorter, still gets the point across. Thoughts? (trying to keep this as small as possible lol)